sale_expected_delivery_date/__openerp__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+#
+#
+#    Author: Kitti U. <kittiu@gmail.com>
+#    Copyright 2015 Ecosoft
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#
+
+{
+    'name': 'Sales Expected Delivery Date',
+    'version': '1.0',
+    'author': "Ecosoft, Odoo Community Association (OCA)",
+    'category': 'Sale',
+    'license': 'AGPL-3',
+    'complexity': 'easy',
+    'images': [],
+    'website': "http://www.ecosoft.co.th",
+    'description': """
+This module provides "Expected Delivery Date" in Sales Order.
```

Comment:
Please extract description into a README.rst file for allowing the view in both GitHub and Odoo, and use OCA's template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst